vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -63,14 +68,45 @@ def from_config(cls, config: Dict[str, Any]) -> "Fp8Config":
         quant_method = cls.get_from_keys(config, ["quant_method"])
         is_checkpoint_fp8_serialized = ("fp8" in quant_method)
         activation_scheme = cls.get_from_keys(config, ["activation_scheme"])
+        ignored_layers = cls.get_from_keys(config, ["ignored_layers"])
         return cls(is_checkpoint_fp8_serialized=is_checkpoint_fp8_serialized,
-                   activation_scheme=activation_scheme)
+                   activation_scheme=activation_scheme,
+                   ignored_layers=ignored_layers)
+
+    def _is_layer_skipped(self, prefix: str) -> bool:
```

Comment:
could this be shared in `quant_utils.py`?

Reply:
I kept it apart since the compressed-tensors logic was sufficiently different. I can combine it for fp8 and fbgemm_fp8